### PR TITLE
[#174] Some of supported options for PDD are not listed in the README.md

### PR DESCRIPTION
**Pull request by Doldrums**
I slightly changed the Readme file and added examples and descriptions for supported PDD options. This is a fix of issue #174

### CODE REVIEW

**Comment by AntistasII** [comment]
@yegor256  please review the PR

File: README.md [hunk 1]
```diff
@@ -41,14 +41,30 @@ Run it locally and read its output:
 $ pdd --help
 ```
 
-You can exclude certain files from the search, for example:
+### File and Directory Selection
```
Comment (yegor256) [inline README.md:44]: @Doldrums maybe it's better to keep the text you just deleted? Now it looks weird: you start a section without any text, just some example of code. It doesn't read well.

**Comment by yegor256** [comment]
@Doldrums see above

File: README.md (hunk 1 above)
Comment (Doldrums) [reply README.md:44]: okay, returned this text 😊

**Comment by Doldrums** [comment]
@yegor256 review please

**Comment by AntistasII** [comment]
@yegor256 Don't forget to review this pull request

**Comment by yegor256** [comment]
@rultor merge

**Comment by Doldrums** [comment]
@yegor256 hey, looks like this is not a problem in PR's source code changes, and actually seems to be just a bug, maybe run @rultor merge again

**Comment by Doldrums** [comment]
@rultor hello

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**